query/deployment.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   selector:
     matchLabels:
       app: query
-  replicas: 1
+  replicas: 3
```

Comment:
The connection the service is a websocket, right?  What happens if you reconnect and get connected to another instance?

Reply:
The client does not support reconnection currently. If you lose your connection while waiting for a job, everything falls apart, currently. Resiliency to connection lost mid-job is on my list to address, but I haven't thought it through currently.

In the meantime, this lets me run the tests quickly because our tests seem to bottlenecked by compiler CPU use. I can saturate >10 query instances at >100% cpu use with a client using 96 processes to run all the Hail Python tests.